components/MashForms/FinalDecisionForm.tsx

```diff
@@ -177,6 +177,7 @@ const FinalDecisionForm = ({
           <a
             href="#"
             className={`lbh-link lbh-link--no-visited-state govuk-!-margin-left-3`}
+            onClick={() => window.history.back()}
```

Comment:
This is a bit of a nitpick, but I _think_ we should use Next.js' `router.back()` method here, instead of `window.history.back()`. It's not a massive deal but might be nicer. Docs for it are here: https://nextjs.org/docs/api-reference/next/router#routerback

Reply:
Will take a look! Thanks

Reply:
Have made the change on the form pages - just need to figure out how to alter the tests